test/_test_data/test_pkg_name_not_in_spdx/package.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>test_pkg_name_not_in_spdx</name>
+  <license file="LICENSE">BSD-3</license>
```

Comment:
```suggestion
  <license file="LICENSE">BSD</license>
```
to match the _old_ ROS approach like e.g. https://github.com/ros/ros_tutorials/blob/db6328879831b4abadac939bd69b43133414c0db/roscpp_tutorials/package.xml#L11